ngrinder-frontend/src/js/components/perftest/detail/Config.vue

```diff
@@ -137,7 +137,7 @@
                              @change="changeScript"
                              @opening="openingScriptSelect"
                              :validationRules="{ required: true, scriptValidation: true }" errStyle="position: absolute; padding-left: 177px;">
-                        <option></option>
+                        <option data-validate="-1"></option>
```

Comment:
This change prevents to `refreshTargetHosts` API call when the script option is empty.